packages/app/app/layout.tsx

```diff
@@ -1,19 +1,16 @@
 import { PropsWithChildren } from "react";
+import { Navbar } from "@/components";
+import { Providers } from "@/providers";
 import "../styles/global.css";
-import Navbar from "@/components/navbar/Navbar";
-
-export const metadata = {
-  title: "Stackly | Stack crypto over time.",
-  description:
-    "Stackly is a simple, non-custodial tool that uses the CoW protocol to place recurring swaps based on DCA..",
-};
```

Comment:
I think this is important to have on this layout file. This gives a default title and description to all dapp. Overridden on the files we want.